spec/lib/webfinger_spec.rb

```diff
@@ -59,11 +59,11 @@
       expect(finger.get(url)).to eq(host_meta_xrd)
     end
 
-    it 'follows redirects' do
+    it "follows redirects" do
       redirect_url = "http://whereami.whatisthis/host-meta"
 
       stub_request(:get, "https://bar.com/.well-known/host-meta").
-        to_return(:status => 302, :headers => { 'Location' => redirect_url })
+        to_return(:status => 302, :headers => { "Location" => redirect_url })
```

Comment:
Really? Well, for starters it should be Ruby 1.9 hash syntax, `to_return status: 302, headers: {"Location" => redirect_url}` is what I'd go for.

But for me pronto too reports:
```
spec/lib/webfinger_spec.rb:66 I: Space inside { detected.
spec/lib/webfinger_spec.rb:66 I: Space inside } detected.
```